keep/action/action.py

```diff
@@ -22,8 +24,13 @@ def __init__(
         self.action_config = config
         self.io_handler = IOHandler()
         self.context_manager = ContextManager.get_instance()
+        self.state_manager = StateManager.get_instance()
 
     def run(self):
+        throttled = self._check_throttling(self.action_config.get("name"))
+        if throttled:
+            self.logger.debug("Action %s is throttled", self.action_config.get("name"))
```

Comment:
Perhaps change to info? w/o verbose I won't know why the alert was not triggered